### ISSUE TITLE
Google Scholar web search not opening results box

### ISSUE THREAD

**Issue by hudcap** [JabRef/jabref#4027]
- [x] I have tested the latest master version from http://builds.jabref.org/master/ and the problem persists

JabRef version 4.2 on Windows 10

Steps to reproduce:

1. Run a web search using Google Scholar
2. The results box pops up and then immediately closes

On the bottom, it says "Status: Processing {search term}"
Other databases work fine.

The event log shows nothing. I'm not sure if there's another log and where that would be.

**Comment by hudcap** [comment]
Apparently this is network-specific.
This issue arose while I was connected to my school's wifi network.
It's working fine on my home network.
Please let me know if there's something I can do to generate a log the next time I'm at school.

**Comment by Siedlerchr** [comment]
Glad to hear that it works. Any errors should be visible in the event log under help

**Comment by jianton** [comment]
I have exactly the same problem, using version 4.2. It started a couple of days ago and the issue persists either using my office network or the home one.

As my jabref was already updated by then and the problem only applies to google scholar, I guess it could be a problem of google scholar...

**Comment by JavierMF** [comment]
I'm getting a 403 (Forbidden) when trying to use this feature. The URL in the logs is:  https://scholar.googleusercontent.com/scholar.bib?q=info:Qe1mJfFRJQIJ:scholar.google.com/&output=citation&scisig=AAGBfm0AAAAAWxgcXyl_iWTFuTRt2F7NMsbZPdAb4JgK&scisf=4&ct=citation&cd=1&hl=en

![captura de pantalla 2018-06-06 a las 19 32 17](https://user-images.githubusercontent.com/3293795/41055165-cc7ca110-69c0-11e8-9108-735036fcdaab.png)

**Comment by Siedlerchr** [comment]
It seems that the export on Google Scholar itself is broken, too.. At least when I manually search for sth and click on any of the buttons I always get the same 403 error.
Could you please verify if it's for you the same?
![grafik](https://user-images.githubusercontent.com/320228/41319366-d45af578-6e9b-11e8-89a4-716048f64518.png)

**Comment by matthiasgeiger** [comment]
Nope. Checking it from a mobile (and therefore of course without having used JabRef before for searching) it works properly. So Google seems to have strictened its 'bot detection' policy again.

**Comment by tobiasdiez** [comment]
I've the same problem (bibtex export always results in 403) and contacted google about it. Let's see what they say.

**Comment by lindenblatt** [comment]
@Siedlerchr
Having this problem at the moment, including 403 when trying to export from Google Scholar.
Proved that I'm not a robot several times, cleared Chrome cache and cookies, reset wireless adapter and router to no avail.

@tobiasdiez
Did you receive any reply from Google?

**Comment by stekeller** [comment]
Having the same issue with the latest JabRef version (4.3.1) on Windows 10.

**Comment by frankwxu** [comment]
It seems only works for the first 10 searches.

**Comment by mattcbro** [comment]
None of the web search features have ever really worked for me for any version of JabRef on any network.  Occasionally google scholar starts randomly working,  but usually not.  In JabRef 5.0 beta I just get a blank screen when it searches.  In prior versions I would get errors or simply crashes.

CiteSeer will bring up articles,  but the titles are completely unrelated to my search words.

**Comment by SengerM** [comment]
I am having the same problem with Google Scholar. When I hit "search" it starts and keeps searching for an infinite amount of time. I have JabRef 5.1 on Ubuntu 20.04.
